shopgate/pwa
# Use currentTarget for the element with the event listener

# Description

This PR resolves the wrong target in #793. Alternatively we could also catch `e` ourself, set the needed value to the one of `currentTarget` and call the original onClick method with `e`.

## Type of change

Please add an "x" into the option that is relevant:


## How to test it

Create any html elements in the button element for the search suggestion item and click on the element. It should correctly get the value of the element with the event listener, not one of its children.
